app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,28 @@
+class LikesController < ApplicationController
+  before_action :authenticate_user!
+
+  def create
+    likeable = Post.find(params[:like][:likeable_id])
```

Comment:
hmm, do you actually need it? You have post data in `like_params`. Make the same approach as in destroy action:
`@like = current_user.likes.build(like_params)`, and you can remove merging user_id in `like_params`